### ISSUE TITLE
Key backup restore dialog overrides settings dialog

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#12230]
It should show up on top of the settings dialog, not replace it

**Comment by jryans** [comment]
I agree this is not great, but I'll going to leave it out of cross-signing scope for now. I view it as a general symptom of modals needing a better stacking system.

**Comment by turt2live** [comment]
It's a boolean we probably removed without realizing.

**Comment by jryans** [comment]
I think it's actually that most of the cross-signing flows are also using `static = true` because the access secret storage dialog may appear on top of them.

That's what I mean, are now finding ourselves possibly stacking up to 3 modals on top and need to preserve state in all of them. 😅

**Comment by turt2live** [comment]
oh... well... ugh.

**Comment by sirocyl** [comment]
I'm having this issue now, and others regarding modal stacking and state loss between them, not being able to select one behind the topmost/most recent modal. Maybe a modal UI isn't the right representation for this kind of event?
![Riot_Ys72P8KybS](https://user-images.githubusercontent.com/944067/80982872-e2d1e100-8df9-11ea-8b1b-4d035c701e11.png)

**Comment by jryans** [comment]
That is a different modal, and it no longer exists in the upcoming 1.6 release (toasts are used instead), so I'd consider that one solved.
